webapp/src/api/main.js

```diff
@@ -0,0 +1,3 @@
+const API_PREFIX = "https://us-central1-ecommerce-eed77.cloudfunctions.net/api"
+
+export default API_PREFIX;
```

Comment:
use and object here to export such items,
you might need to export another variable and then this'll create or a problem.
or
another possible solution is to create environment variables.